package/linux/make-package

```diff
@@ -225,6 +225,20 @@ if [ -n "${JENKINS_URL}" ]; then
    printenv
 fi
 
+# if SCCACHE_ENABLED environment variable is set, use sccache
+if [ -n "$SCCACHE_ENABLED" ]; then
+    echo "Using sccache"
+   
+    if [ -n "$AWS_ACCESS_KEY_ID" ] || [ $(aws sts get-caller-identity --query "Account" --profile ${AWS_PROFILE:-sso}) -eq 14 ]; then
```

Comment:
Do any of these need to be pre-defined before this script runs? If so we'll want to add that info into the wiki